aten/src/ATen/native/quantized/cuda/fake_quantize_core.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
I think this file should be called `.cuh`

Reply:
No, it is not necessary. We should use cuh only when the file itself uses cuda primitives. In this case, the file does not do that .